pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "wts"
-version = "0.3.2"
+version = "0.3.3"
```

Comment:
I think this `toml` file version needs updating. The last released version of WTS is 0.4.0, see https://github.com/uc-cdis/workspace-token-service/releases/tag/0.4.0

Reply:
oh you're right...